src/preferences/credentials.cpp

```diff
@@ -230,21 +233,15 @@ namespace preferences
 			filesystem::delete_file(filesystem::get_credentials_file());
 			return;
 		}
-		secure_buffer credentials_data(1, CREDENTIAL_SEPARATOR);
-		std::size_t offset = 1;
+		secure_buffer credentials_data(0, CREDENTIAL_SEPARATOR);
```

Comment:
`(0, CREDENTIAL_SEPARATOR)` does nothing - just remove it.

Reply:
Hey, I was expecting that -- I just newb enough in C++ to not know the alternative.. So what do I write then, `credentials_data();` or `credentials_data;` ?

Reply:
UPD: neverming, found it, the compiler is pretty unambiguous on what I should do anyway :sweat_smile:

Reply:
Thanks for the review! Addressed the comment